src/wwwroot/partials/footer.html

```diff
@@ -1,8 +1,8 @@
 <div class="section--wrapper">
-  <p>
+  <p white-brand-toggler-text>
```

Comment:
What does it mean in an empty `p`? how it works?

Reply:
Tt's not an empty P, it's used as a container for an svg, text and also a link.

Reply:
But, the text is inside another tag with _white-brand-toggler-text_ attribute, right? So, is it necessary here too?